### PR TITLE
Added SignUp

### PR DESCRIPTION
**Pull request by Lazer20**
Added Sign Up

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>signup</title>
+</head>
+<body>
+    <p>name</p>
+    <p>emqil</p>
```
Comment (kzngoga) [inline signup.html]: aha wayirashe ni 'email'

**Comment by kzngoga** [comment]
great job puta!

**Comment by Lazer20** [comment]
sawa mapumbu

File: signup.html (hunk 1 above)
Comment (Lazer20) [reply signup.html]: sawa sawa

**Comment by Lazer20** [comment]
huuhuhuhuhuhuhu

**Pull request merged**